src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.cpp

```diff
@@ -1035,3 +1035,44 @@ void runtime::intelgpu::do_convert_operation(cldnn::topology& topology,
                                                  gws);
     topology.add(op_convert);
 }
+
+void runtime::intelgpu::do_sigmoid_backprop_operation(cldnn::topology& topology,
+                                                      const string& input_name,
+                                                      const Shape& input_shape,
+                                                      const string& delta_name,
+                                                      const Shape& delta_shape,
+                                                      const string& output_name,
+                                                      const Shape& output_shape,
+                                                      const element::Type& output_type)
+{
+    const string entry_point_name = "op_sigmoid_backprop_" + output_name;
+    codegen::CodeWriter writer;
+    vector<size_t> gws;
+
+    runtime::intelgpu::gen_func_def(
```

Comment:
No need to use `runtime::intelgpu::` in case of the function is inside the namespace.